english/hammertime/hammertime.md

```diff
@@ -0,0 +1,2 @@
+You break into the hammertime-dance, but quickly realize that this
+doesn't really get to [anywhere](english/marshmallow.md) 
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.